scripts/pylint_extensions_test.py

```diff
@@ -204,8 +204,8 @@ def test_finds_backslash_continuation(self):
 
         with open(filename, 'w') as tmp:
             tmp.write(
-                """message1 = 'abc'\\\n""" # pylint: disable=backslash-continuation
-                """'cde'\\\n""" # pylint: disable=backslash-continuation
+                """message1 = 'abc'\\\n"""
```

Comment:
why this change?

Reply:
its someone's change, i do it because its giving lints issue